maven-surefire-plugin empty groups configuration

Configuring maven-surefire-plugin with an empty groups argument like this:

```
				<plugin>
					<groupId>org.apache.maven.plugins</groupId>
					<artifactId>maven-surefire-plugin</artifactId>
					<version>2.22.2</version>
					<configuration>
						<groups></groups>
						<excludedGroups>slowTests</excludedGroups>
					</configuration>
				</plugin>
```

cause an exception

In maven-surefire-plugin an empty groupd argument is handled here:
https://github.com/apache/maven-surefire/blob/7c4faa9ae6fb6e15771753cbce7fb99fa669af41/surefire-providers/surefire-testng/src/main/java/org/apache/maven/surefire/testng/TestNGReporter.java#L155

For pitest-junit5-plugin the exception is not handled on this line: https://github.com/pitest/pitest-junit5-plugin/blob/7f563c9466cc5221f9f462be9d40956d83302e68/src/main/java/org/pitest/junit5/JUnit5TestUnitFinder.java#LL76C29-L76C38

I think this exception should be handled, as i think it should be handled like in maven-surefire-plugin as it use the setting from it.

here is the actual exception with all around it:

```
10:43:23 PIT >> INFO : ---------------------------------------------------------------------------
10:43:23 PIT >> FINE : Running report with ReportOptions[targetClasses=[works.reload.recore.intl.*, works.reload.recore.logger.*, works.reload.recore.util.*, works.reload.recore.context.*, works.reload.recore.task.*], excludedMethods=[], excludedClasses=[], excludedTestClasses=[], codePaths=[C:\Users\Gabriel\IdeaProjects\ReBuild\ReCore\target\classes], reportDir='C:\Users\Gabriel\IdeaProjects\ReBuild\ReCore\target\pit-reports', historyInputLocation=null, historyOutputLocation=null, sourceDirs=[C:\Users\Gabriel\IdeaProjects\ReBuild\ReCore\src\main\java, C:\Users\Gabriel\IdeaProjects\ReBuild\ReCore\target\generated-sources\annotations, C:\Users\Gabriel\IdeaProjects\ReBuild\ReCore\src\test\java, C:\Users\Gabriel\IdeaProjects\ReBuild\ReCore\target\generated-test-sources\test-annotations], classPathElements=[C:\Users\Gabriel\IdeaProjects\ReBuild\ReCore\target\test-classes, C:\Users\Gabriel\IdeaProjects\ReBuild\ReCore\target\classes, C:\Users\Gabriel\.m2\repository\org\jetbrains\annotations\24.0.1\annotations-24.0.1.jar, C:\Users\Gabriel\.m2\repository\org\junit\jupiter\junit-jupiter-engine\5.10.0-M1\junit-jupiter-engine-5.10.0-M1.jar, C:\Users\Gabriel\.m2\repository\org\junit\platform\junit-platform-engine\1.10.0-M1\junit-platform-engine-1.10.0-M1.jar, C:\Users\Gabriel\.m2\repository\org\opentest4j\opentest4j\1.2.0\opentest4j-1.2.0.jar, C:\Users\Gabriel\.m2\repository\org\junit\platform\junit-platform-commons\1.10.0-M1\junit-platform-commons-1.10.0-M1.jar, C:\Users\Gabriel\.m2\repository\org\junit\jupiter\junit-jupiter-api\5.10.0-M1\junit-jupiter-api-5.10.0-M1.jar, C:\Users\Gabriel\.m2\repository\org\apiguardian\apiguardian-api\1.1.2\apiguardian-api-1.1.2.jar, C:\Users\Gabriel\.m2\repository\org\junit\jupiter\junit-jupiter-params\5.10.0-M1\junit-jupiter-params-5.10.0-M1.jar, C:\Users\Gabriel\.m2\repository\org\junit\platform\junit-platform-launcher\1.10.0-M1\junit-platform-launcher-1.10.0-M1.jar, C:\Users\Gabriel\.m2\repository\org\hamcrest\hamcrest\2.2\hamcrest-2.2.jar, C:\Users\Gabriel\.m2\repository\org\mockito\mockito-core\5.3.1\mockito-core-5.3.1.jar, C:\Users\Gabriel\.m2\repository\net\bytebuddy\byte-buddy\1.14.4\byte-buddy-1.14.4.jar, C:\Users\Gabriel\.m2\repository\net\bytebuddy\byte-buddy-agent\1.14.4\byte-buddy-agent-1.14.4.jar, C:\Users\Gabriel\.m2\repository\org\objenesis\objenesis\3.3\objenesis-3.3.jar, C:\Users\Gabriel\.m2\repository\org\mockito\mockito-junit-jupiter\5.3.1\mockito-junit-jupiter-5.3.1.jar, C:\Users\Gabriel\.m2\repository\com\github\spotbugs\spotbugs-annotations\4.7.3\spotbugs-annotations-4.7.3.jar, C:\Users\Gabriel\.m2\repository\com\google\code\findbugs\jsr305\3.0.2\jsr305-3.0.2.jar, C:\Users\Gabriel\.m2\repository\org\pitest\pitest-junit5-plugin\1.2.0\pitest-junit5-plugin-1.2.0.jar, C:\Users\Gabriel\.m2\repository\org\pitest\pitest\1.10.3\pitest-1.10.3.jar], mutators=[], features=[], jvmArgs=[-Djava.awt.headless=true], argLine=null, numberOfThreads=1, timeoutFactor=1.25, timeoutConstant=3000, targetTests=[^works\.reload\.recore\.intl\..*$, ^works\.reload\.recore\.util\..*$, ^works\.reload\.recore\.task\..*$], loggingClasses=[], verbosity=VERBOSE, failWhenNoMutations=true, skipFailingTests=false, outputs=[HTML], groupConfig=TestGroupConfig [excludedGroups=[slow], includedGroups=[]], fullMutationMatrix=false, mutationUnitSize=0, shouldCreateTimestampedReports=false, detectInlinedCode=true, exportLineCoverage=false, mutationThreshold=0, coverageThreshold=0, testStrengthThreshold=0, mutationEngine='gregor', javaExecutable='null', includeLaunchClasspath=true, properties={}, maxSurvivors=0, excludedRunners=[], includedTestMethods=[], testPlugin='', useClasspathJar=false, projectBase=C:\Users\Gabriel\IdeaProjects\ReBuild, inputEncoding=UTF-8, outputEncoding=UTF-8]
10:43:23 PIT >> FINE : System class path is C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3\boot\plexus-classworlds-2.6.0.jar;C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3\boot\plexus-classworlds.license
10:43:23 PIT >> FINE : Maximum available memory is 8120 mb
10:43:23 PIT >> FINE : Incremental analysis set 570 mutations to a status of NOT_STARTED
10:43:23 PIT >> INFO : Incremental analysis reduced number of mutations by 0
10:43:23 PIT >> INFO : Created 24 mutation test units in pre scan
10:43:24 PIT >> FINE : MINION : Installing PIT agent
10:43:24 PIT >> INFO : Sending 36 test classes to minion
10:43:24 PIT >> INFO : Sent tests to minion
10:43:24 PIT >> INFO : MINION : 10:43:24 PIT >> FINE : Expecting 36 tests classes from parent
10:43:24 PIT >> INFO : MINION : 10:43:24 PIT >> FINE : Tests classes received
10:43:24 PIT >> INFO : MINION : 10:43:24 PIT >> INFO : Checking environment
10:43:24 PIT >> FINE : MINION : java.lang.IllegalArgumentException: Error creating tag filter
10:43:24 PIT >> FINE : MINION : 	at org.pitest.junit5.JUnit5TestUnitFinder.findTestUnits(JUnit5TestUnitFinder.java:79)
10:43:24 PIT >> FINE : MINION : 	at org.pitest.mutationtest.config.PrioritisingTestUnitFinder.findTestUnits(PrioritisingTestUnitFinder.java:20)
10:43:24 PIT >> FINE : MINION : 	at org.pitest.testapi.execute.FindTestUnits.findTestUnits(FindTestUnits.java:64)
10:43:24 PIT >> FINE : MINION : 	at org.pitest.testapi.execute.FindTestUnits.getTestUnits(FindTestUnits.java:47)
10:43:24 PIT >> FINE : MINION : 	at org.pitest.testapi.execute.FindTestUnits.findTestUnitsForAllSuppliedClasses(FindTestUnits.java:36)
10:43:24 PIT >> FINE : MINION : 	at org.pitest.coverage.execute.CoverageMinion.discoverTests(CoverageMinion.java:167)
10:43:24 PIT >> FINE : MINION : 	at org.pitest.coverage.execute.CoverageMinion.getTestsFromParent(CoverageMinion.java:153)
10:43:24 PIT >> FINE : MINION : 	at org.pitest.coverage.execute.CoverageMinion.main(CoverageMinion.java:84)
10:43:24 PIT >> FINE : MINION : Caused by: org.junit.platform.commons.PreconditionViolationException: Unable to parse tag expression "": empty tag expression
10:43:24 PIT >> FINE : MINION : 	at org.junit.platform.launcher.TagFilter.lambda$parse$8(TagFilter.java:181)
10:43:24 PIT >> FINE : MINION : 	at org.junit.platform.launcher.tagexpression.ParseResult.tagExpressionOrThrow(ParseResult.java:42)
10:43:24 PIT >> FINE : MINION : 	at org.junit.platform.launcher.TagFilter.parse(TagFilter.java:180)
10:43:24 PIT >> FINE : MINION : 	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
10:43:24 PIT >> FINE : MINION : 	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:992)
10:43:24 PIT >> FINE : MINION : 	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
10:43:24 PIT >> FINE : MINION : 	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
10:43:24 PIT >> FINE : MINION : 	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
10:43:24 PIT >> FINE : MINION : 	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
10:43:24 PIT >> FINE : MINION : 	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
10:43:24 PIT >> FINE : MINION : 	at org.junit.platform.launcher.TagFilter.parseAll(TagFilter.java:176)
10:43:24 PIT >> FINE : MINION : 	at org.junit.platform.launcher.TagFilter.includeMatching(TagFilter.java:133)
10:43:24 PIT >> FINE : MINION : 	at org.junit.platform.launcher.TagFilter.includeTags(TagFilter.java:89)
10:43:24 PIT >> FINE : MINION : 	at org.pitest.junit5.JUnit5TestUnitFinder.findTestUnits(JUnit5TestUnitFinder.java:76)
10:43:24 PIT >> FINE : MINION : 	... 7 more
10:43:24 PIT >> INFO : MINION : 10:43:24 PIT >> SEVERE : Error calculating coverage. Process will exit.
10:43:24 PIT >> INFO : MINION : java.lang.IllegalArgumentException: Error creating tag filter
10:43:24 PIT >> INFO : MINION : 	at org.pitest.junit5.JUnit5TestUnitFinder.findTestUnits(JUnit5TestUnitFinder.java:79)
10:43:24 PIT >> INFO : MINION : 	at org.pitest.mutationtest.config.PrioritisingTestUnitFinder.findTestUnits(PrioritisingTestUnitFinder.java:20)
10:43:24 PIT >> INFO : MINION : 	at org.pitest.testapi.execute.FindTestUnits.findTestUnits(FindTestUnits.java:64)
10:43:24 PIT >> INFO : MINION : 	at org.pitest.testapi.execute.FindTestUnits.getTestUnits(FindTestUnits.java:47)
10:43:24 PIT >> INFO : MINION : 	at org.pitest.testapi.execute.FindTestUnits.findTestUnitsForAllSuppliedClasses(FindTestUnits.java:36)
10:43:24 PIT >> INFO : MINION : 	at org.pitest.coverage.execute.CoverageMinion.discoverTests(CoverageMinion.java:167)
10:43:24 PIT >> INFO : MINION : 	at org.pitest.coverage.execute.CoverageMinion.getTestsFromParent(CoverageMinion.java:153)
10:43:24 PIT >> INFO : MINION : 	at org.pitest.coverage.execute.CoverageMinion.main(CoverageMinion.java:84)
10:43:24 PIT >> INFO : MINION : Caused by: org.junit.platform.commons.PreconditionViolationException: Unable to parse tag expression "": empty tag expression
10:43:24 PIT >> INFO : MINION : 	at org.junit.platform.launcher.TagFilter.lambda$parse$8(TagFilter.java:181)
10:43:24 PIT >> INFO : MINION : 	at org.junit.platform.launcher.tagexpression.ParseResult.tagExpressionOrThrow(ParseResult.java:42)
10:43:24 PIT >> INFO : MINION : 	at org.junit.platform.launcher.TagFilter.parse(TagFilter.java:180)
10:43:24 PIT >> INFO : MINION : 	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
10:43:24 PIT >> INFO : MINION : 	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:992)
10:43:24 PIT >> INFO : MINION : 	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
10:43:24 PIT >> INFO : MINION : 	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
10:43:24 PIT >> INFO : MINION : 	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
10:43:24 PIT >> INFO : MINION : 	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
10:43:24 PIT >> INFO : MINION : 	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
10:43:24 PIT >> INFO : MINION : 	at org.junit.platform.launcher.TagFilter.parseAll(TagFilter.java:176)
10:43:24 PIT >> INFO : MINION : 	at org.junit.platform.launcher.TagFilter.includeMatching(TagFilter.java:133)
10:43:24 PIT >> INFO : MINION : 	at org.junit.platform.launcher.TagFilter.includeTags(TagFilter.java:89)
10:43:24 PIT >> INFO : MINION : 	at org.pitest.junit5.JUnit5TestUnitFinder.findTestUnits(JUnit5TestUnitFinder.java:76)
10:43:24 PIT >> INFO : MINION : 	... 7 more
10:43:24 PIT >> SEVERE : Coverage generator Minion exited abnormally due to UNKNOWN_ERROR
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] SubModules ............................... SUCCESS [  1.472 s]
[INFO] SubModules ............................... SUCCESS [  1.035 s]
[INFO] SubModules ............................... FAILURE [  0.805 s]
[INFO] SubModules ............................... SKIPPED
[INFO] SubModules ............................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.496 s
[INFO] Finished at: 2023-05-30T10:43:24+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.pitest:pitest-maven:1.10.3:mutationCoverage (default-cli) on project recore: Execution default-cli of goal org.pitest:pitest-maven:1.10.3:mutationCoverage failed: Coverage generation minion exited abnormally! (UNKNOWN_ERROR)
[ERROR] 
[ERROR] Please copy and paste the information and the complete stacktrace below when reporting an issue
[ERROR] VM : Java HotSpot(TM) 64-Bit Server VM
[ERROR] Vendor : Oracle Corporation
[ERROR] Version : 17.0.4.1+1-LTS-2
[ERROR] Uptime : 4377
[ERROR] Input -> 
[ERROR]  1 : -Dmaven.multiModuleProjectDirectory=C:\Users\Gabriel\IdeaProjects\ReBuild
[ERROR]  2 : -Djansi.passthrough=true
[ERROR]  3 : -Dmaven.home=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3
[ERROR]  4 : -Dclassworlds.conf=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3\bin\m2.conf
[ERROR]  5 : -Dmaven.ext.class.path=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven-event-listener.jar
[ERROR]  6 : -javaagent:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\lib\idea_rt.jar=54493:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\bin
[ERROR]  7 : -Dfile.encoding=UTF-8
[ERROR] BootClassPathSupported : false
[ERROR] 
[ERROR] 
[ERROR] Please copy and paste the information and the complete stacktrace below when reporting an issue
[ERROR] VM : Java HotSpot(TM) 64-Bit Server VM
[ERROR] Vendor : Oracle Corporation
[ERROR] Version : 17.0.4.1+1-LTS-2
[ERROR] Uptime : 4377
[ERROR] Input -> 
[ERROR]  1 : -Dmaven.multiModuleProjectDirectory=C:\Users\Gabriel\IdeaProjects\ReBuild
[ERROR]  2 : -Djansi.passthrough=true
[ERROR]  3 : -Dmaven.home=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3
[ERROR]  4 : -Dclassworlds.conf=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3\bin\m2.conf
[ERROR]  5 : -Dmaven.ext.class.path=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven-event-listener.jar
[ERROR]  6 : -javaagent:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\lib\idea_rt.jar=54493:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\bin
[ERROR]  7 : -Dfile.encoding=UTF-8
[ERROR] BootClassPathSupported : false
[ERROR] 
[ERROR] -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.pitest:pitest-maven:1.10.3:mutationCoverage (default-cli) on project recore: Execution default-cli of goal org.pitest:pitest-maven:1.10.3:mutationCoverage failed: Coverage generation minion exited abnormally! (UNKNOWN_ERROR)

Please copy and paste the information and the complete stacktrace below when reporting an issue
VM : Java HotSpot(TM) 64-Bit Server VM
Vendor : Oracle Corporation
Version : 17.0.4.1+1-LTS-2
Uptime : 4377
Input -> 
 1 : -Dmaven.multiModuleProjectDirectory=C:\Users\Gabriel\IdeaProjects\ReBuild
 2 : -Djansi.passthrough=true
 3 : -Dmaven.home=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3
 4 : -Dclassworlds.conf=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3\bin\m2.conf
 5 : -Dmaven.ext.class.path=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven-event-listener.jar
 6 : -javaagent:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\lib\idea_rt.jar=54493:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\bin
 7 : -Dfile.encoding=UTF-8
BootClassPathSupported : false


Please copy and paste the information and the complete stacktrace below when reporting an issue
VM : Java HotSpot(TM) 64-Bit Server VM
Vendor : Oracle Corporation
Version : 17.0.4.1+1-LTS-2
Uptime : 4377
Input -> 
 1 : -Dmaven.multiModuleProjectDirectory=C:\Users\Gabriel\IdeaProjects\ReBuild
 2 : -Djansi.passthrough=true
 3 : -Dmaven.home=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3
 4 : -Dclassworlds.conf=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3\bin\m2.conf
 5 : -Dmaven.ext.class.path=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven-event-listener.jar
 6 : -javaagent:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\lib\idea_rt.jar=54493:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\bin
 7 : -Dfile.encoding=UTF-8
BootClassPathSupported : false

    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at org.codehaus.classworlds.Launcher.main (Launcher.java:47)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution default-cli of goal org.pitest:pitest-maven:1.10.3:mutationCoverage failed: Coverage generation minion exited abnormally! (UNKNOWN_ERROR)

Please copy and paste the information and the complete stacktrace below when reporting an issue
VM : Java HotSpot(TM) 64-Bit Server VM
Vendor : Oracle Corporation
Version : 17.0.4.1+1-LTS-2
Uptime : 4377
Input -> 
 1 : -Dmaven.multiModuleProjectDirectory=C:\Users\Gabriel\IdeaProjects\ReBuild
 2 : -Djansi.passthrough=true
 3 : -Dmaven.home=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3
 4 : -Dclassworlds.conf=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3\bin\m2.conf
 5 : -Dmaven.ext.class.path=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven-event-listener.jar
 6 : -javaagent:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\lib\idea_rt.jar=54493:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\bin
 7 : -Dfile.encoding=UTF-8
BootClassPathSupported : false


Please copy and paste the information and the complete stacktrace below when reporting an issue
VM : Java HotSpot(TM) 64-Bit Server VM
Vendor : Oracle Corporation
Version : 17.0.4.1+1-LTS-2
Uptime : 4377
Input -> 
 1 : -Dmaven.multiModuleProjectDirectory=C:\Users\Gabriel\IdeaProjects\ReBuild
 2 : -Djansi.passthrough=true
 3 : -Dmaven.home=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3
 4 : -Dclassworlds.conf=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3\bin\m2.conf
 5 : -Dmaven.ext.class.path=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven-event-listener.jar
 6 : -javaagent:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\lib\idea_rt.jar=54493:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\bin
 7 : -Dfile.encoding=UTF-8
BootClassPathSupported : false

    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:148)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at org.codehaus.classworlds.Launcher.main (Launcher.java:47)
Caused by: org.pitest.util.PitError: Coverage generation minion exited abnormally! (UNKNOWN_ERROR)

Please copy and paste the information and the complete stacktrace below when reporting an issue
VM : Java HotSpot(TM) 64-Bit Server VM
Vendor : Oracle Corporation
Version : 17.0.4.1+1-LTS-2
Uptime : 4377
Input -> 
 1 : -Dmaven.multiModuleProjectDirectory=C:\Users\Gabriel\IdeaProjects\ReBuild
 2 : -Djansi.passthrough=true
 3 : -Dmaven.home=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3
 4 : -Dclassworlds.conf=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3\bin\m2.conf
 5 : -Dmaven.ext.class.path=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven-event-listener.jar
 6 : -javaagent:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\lib\idea_rt.jar=54493:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\bin
 7 : -Dfile.encoding=UTF-8
BootClassPathSupported : false


Please copy and paste the information and the complete stacktrace below when reporting an issue
VM : Java HotSpot(TM) 64-Bit Server VM
Vendor : Oracle Corporation
Version : 17.0.4.1+1-LTS-2
Uptime : 4377
Input -> 
 1 : -Dmaven.multiModuleProjectDirectory=C:\Users\Gabriel\IdeaProjects\ReBuild
 2 : -Djansi.passthrough=true
 3 : -Dmaven.home=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3
 4 : -Dclassworlds.conf=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3\bin\m2.conf
 5 : -Dmaven.ext.class.path=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven-event-listener.jar
 6 : -javaagent:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\lib\idea_rt.jar=54493:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\bin
 7 : -Dfile.encoding=UTF-8
BootClassPathSupported : false

    at org.pitest.util.Unchecked.translateCheckedException (Unchecked.java:20)
    at org.pitest.coverage.execute.DefaultCoverageGenerator.calculateCoverage (DefaultCoverageGenerator.java:110)
    at org.pitest.coverage.execute.DefaultCoverageGenerator.calculateCoverage (DefaultCoverageGenerator.java:52)
    at org.pitest.mutationtest.tooling.MutationCoverage.runAnalysis (MutationCoverage.java:149)
    at org.pitest.mutationtest.tooling.MutationCoverage.runReport (MutationCoverage.java:139)
    at org.pitest.mutationtest.tooling.EntryPoint.execute (EntryPoint.java:129)
    at org.pitest.mutationtest.tooling.EntryPoint.execute (EntryPoint.java:57)
    at org.pitest.maven.RunPitStrategy.execute (RunPitStrategy.java:35)
    at org.pitest.maven.AbstractPitMojo.analyse (AbstractPitMojo.java:502)
    at org.pitest.maven.AbstractPitMojo.execute (AbstractPitMojo.java:432)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at org.codehaus.classworlds.Launcher.main (Launcher.java:47)
Caused by: org.pitest.util.PitError: Coverage generation minion exited abnormally! (UNKNOWN_ERROR)

Please copy and paste the information and the complete stacktrace below when reporting an issue
VM : Java HotSpot(TM) 64-Bit Server VM
Vendor : Oracle Corporation
Version : 17.0.4.1+1-LTS-2
Uptime : 4377
Input -> 
 1 : -Dmaven.multiModuleProjectDirectory=C:\Users\Gabriel\IdeaProjects\ReBuild
 2 : -Djansi.passthrough=true
 3 : -Dmaven.home=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3
 4 : -Dclassworlds.conf=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven3\bin\m2.conf
 5 : -Dmaven.ext.class.path=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\plugins\maven\lib\maven-event-listener.jar
 6 : -javaagent:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\lib\idea_rt.jar=54493:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2.1\bin
 7 : -Dfile.encoding=UTF-8
BootClassPathSupported : false

    at org.pitest.coverage.execute.DefaultCoverageGenerator.gatherCoverageData (DefaultCoverageGenerator.java:149)
    at org.pitest.coverage.execute.DefaultCoverageGenerator.calculateCoverage (DefaultCoverageGenerator.java:94)
    at org.pitest.coverage.execute.DefaultCoverageGenerator.calculateCoverage (DefaultCoverageGenerator.java:52)
    at org.pitest.mutationtest.tooling.MutationCoverage.runAnalysis (MutationCoverage.java:149)
    at org.pitest.mutationtest.tooling.MutationCoverage.runReport (MutationCoverage.java:139)
    at org.pitest.mutationtest.tooling.EntryPoint.execute (EntryPoint.java:129)
    at org.pitest.mutationtest.tooling.EntryPoint.execute (EntryPoint.java:57)
    at org.pitest.maven.RunPitStrategy.execute (RunPitStrategy.java:35)
    at org.pitest.maven.AbstractPitMojo.analyse (AbstractPitMojo.java:502)
    at org.pitest.maven.AbstractPitMojo.execute (AbstractPitMojo.java:432)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at org.codehaus.classworlds.Launcher.main (Launcher.java:47)
[ERROR] 
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :recore

Process finished with exit code 1
```

--- a/src/main/java/org/pitest/junit5/JUnit5TestUnitFinder.java
+++ b/src/main/java/org/pitest/junit5/JUnit5TestUnitFinder.java
@@ -17,6 +17,7 @@ package org.pitest.junit5;
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.List;
+import java.util.stream.Collectors;
 
 import static java.util.Collections.emptyList;
 import static java.util.Collections.synchronizedList;
@@ -66,13 +67,13 @@ public class JUnit5TestUnitFinder implements TestUnitFinder {
 
         List<Filter> filters = new ArrayList<>(2);
         try {
-            List<String> excludedGroups = testGroupConfig.getExcludedGroups();
-            if(excludedGroups != null && !excludedGroups.isEmpty()) {
+            List<String> excludedGroups = testGroupConfig.getExcludedGroups().stream().filter(group -> !group.isEmpty()).collect(Collectors.toList());
+            if(!excludedGroups.isEmpty()) {
                 filters.add(TagFilter.excludeTags(excludedGroups));
             }
 
-            List<String> includedGroups = testGroupConfig.getIncludedGroups();
-            if(includedGroups != null && !includedGroups.isEmpty()) {
+            List<String> includedGroups = testGroupConfig.getIncludedGroups().stream().filter(group -> !group.isEmpty()).collect(Collectors.toList());
+            if(!includedGroups.isEmpty()) {
                 filters.add(TagFilter.includeTags(includedGroups));
             }
         } catch(PreconditionViolationException e) {

--- a/src/test/java/org/pitest/junit5/JUnit5TestUnitFinderTest.java
+++ b/src/test/java/org/pitest/junit5/JUnit5TestUnitFinderTest.java
@@ -265,6 +265,12 @@ class JUnit5TestUnitFinderTest {
         findsAndRunsNTests(3, new JUnit5TestUnitFinder(new TestGroupConfig().withExcludedGroups("excluded"), emptyList()), TestClassWithTags.class);
     }
 
+    @Test
+    void excludesIsEmpty() {
+        findsAndRunsNTests(4, new JUnit5TestUnitFinder(new TestGroupConfig().withExcludedGroups(""), emptyList()), TestClassWithTags.class);
+    }
+
+    @Test
     void excludesSpockTestsByTag() {
         findsAndRunsNTests(3, new JUnit5TestUnitFinder(new TestGroupConfig().withExcludedGroups("excluded"), emptyList()), TestSpecWithTags.class);
     }
@@ -272,6 +278,11 @@ class JUnit5TestUnitFinderTest {
     @Test
     void includesTestsByTag() {
         findsAndRunsNTests(1, new JUnit5TestUnitFinder(new TestGroupConfig().withIncludedGroups("included"), emptyList()), TestClassWithTags.class);
+    }
+
+    @Test
+    void includesIsEmpty() {
+        findsAndRunsNTests(4, new JUnit5TestUnitFinder(new TestGroupConfig().withIncludedGroups(""), emptyList()), TestClassWithTags.class);
     }
 
     @Test

